.github/workflows/build.yml

```diff
@@ -21,9 +21,13 @@ jobs:
       TEST_RESULT_DIR: drop\testresults
     steps:
       - uses: actions/checkout@v1
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '9.0.x'
       - name: Build Everything
         shell: powershell
-        run: ./build.ps1
+        run: dotnet build code.sln
```

Comment:
I did a bit of experimentation and, unfortunately, I don't think we can use the code.sln just yet. I couldn't get it to load the one .nuproj we still need for now, the SF.ActorsServices.Internal, or to build it.

Instead, we should be able to `dotnet build buildAll.proj` for now and delete the build.ps1.

Reply:
Changed, although we might not be able to delete `build.ps1` just yet, since it is still being used by the internal build pipeline.